testing/postgres/integration_test.go

```diff
@@ -165,6 +165,29 @@ func TestIntegration_PGDUMP_Command(t *testing.T) {
 	checkResults(t, dbPath)
 }
 
+func TestIntegration_PGDUMP_SchemaCommand(t *testing.T) {
+	t.Parallel()
+
+	tmpdir := prepareIntegrationTest(t)
+	defer os.RemoveAll(tmpdir)
+
+	dataFilepath := "../../test_data/pg_dump.test.out"
+	// Be aware that when testing with the command, the time `now` might be
+	// different between file prefixes and the contents in the files. This
```

Comment:
Offtopic, but as part of integration tests, should we also check if the 4 files are written during the process. We can specify the dbName param in these commands to avoid the name having a random hash and check if the files exist.

Reply:
I agree that we should validate the output produced by schema subcommand as well as actually look at the schema produced to make sure it's what we expect. But to do that we will need to have a golden output for schema generated, to prevent any regression. How about we do this in a follow up PR? (We will though set `prefix` since dbName won't be available for schema subcommand).

Reply:
Sounds good!